tests/phpunit/bin/install-wp-tests.sh

```diff
@@ -166,7 +166,7 @@ install_db() {
 	fi
 
 	# create database
-	if [ $(mysql --user="$DB_USER" --password="$DB_PASS" --execute='show databases;' | grep ^$DB_NAME$) ]
+	if [ $(mysql --user="$DB_USER" --password="$DB_PASS"$EXTRA --execute='show databases;' | grep ^$DB_NAME$) ]
```

Comment:
What's up with this addition? Is it from testing? I just don't see it mentioned elsewhere. 🤔 

```suggestion
	if [ $(mysql --user="$DB_USER" --password="$DB_PASS" --execute='show databases;' | grep ^$DB_NAME$) ]
```

Reply:
There could be weird stuff in there, it should be an exact copy of this: https://raw.githubusercontent.com/wp-cli/scaffold-command/master/templates/install-wp-tests.sh

I literally just dumped in the latest version of the file here 😄

Reply:
Hah, okay cool. Works for me!